src/main/scala/strawman/collection/immutable/Seq.scala

```diff
@@ -80,6 +80,55 @@ trait SeqOps[+A, +CC[_], +C] extends collection.SeqOps[A, CC, C] {
     */
   @`inline` final def :+ [B >: A](elem: B): CC[B] = append(elem)
 
+  /** Returns a new $coll containing the elements from the left hand operand followed by the elements from the
+    *  right hand operand. The element type of the $coll is the most specific superclass encompassing
+    *  the element types of the two operands.
+    *
+    *  @param suffix the traversable to append.
+    *  @tparam B     the element type of the returned collection.
+    *  @return       a new collection of type `CC[B]` which contains all elements
+    *                of this $coll followed by all elements of `suffix`.
+    */
+  def appendAll[B >: A](suffix: collection.Iterable[B]): CC[B] = fromIterable(View.Concat(toIterable, suffix))
```

Comment:
I would make that one `final` and forward to `concat` so that subclasses override only one method (`concat`) and get the benefit of it regardless that they use `++` or `:++`.

Reply:
I pushed df3a9aa which makes `concat` an alias of `appendAll` in `immutable.Seq`. This way any inheritors of `immutable.Seq` can override `prependAll` and `appendAll` if needed for performance (instead of `prependAll` and `concat` which would have been the case if `appendAll` was instead made an alias of `concat`).